content/tutorial/tutorial.md

```diff
@@ -1007,10 +1007,10 @@ Let's `map` over the `history` in the Game's `render` method:
     const current = history[history.length - 1];
     const winner = calculateWinner(current.squares);
 
-    const moves = history.map((step, move) => {
-      const desc = move ?
-        'Go to move #' + move :
-        'Go to game start';
+    const moves = history.map((_, move) => {
```

Comment:
How about to leave the name first parameter (`step`) as it was before?

Reply:
Sounds good, I'd like to suggest you also add a prefix `_` to it (`_step`) as jslint suggestion, which may act as a good guideline to audience that implies the first parameter is unused.

Reply:
Let's leave it for now. Okay?

Reply:
At your disposal.